Sofa/Component/Topology/Mapping/src/sofa/component/topology/mapping/Edge2QuadTopologicalMapping.cpp

```diff
@@ -154,14 +152,14 @@ void Edge2QuadTopologicalMapping::init()
                 {
                     unsigned int p0=i;
 
-                    Mat rotation;
+                    Mat3x3 rotation;
                     (from_mstate->read(core::ConstVecCoordId::position())->getValue())[p0].writeRotationMatrix(rotation);
 
-                    Vec t;
+                    Vec3 t;
                     t=(from_mstate->read(core::ConstVecCoordId::position())->getValue())[p0].getCenter();
```

Comment:
NOT this PR, but FYI in a later PR: https://github.com/sofa-framework/sofa/issues/3363
to store the from_mstate's  positions vector ref out of the loop.
(and those two lines are ugly anyway)

Reply:
This will be totally deleted (in a futur PR). My plan is to keep this mapping as a "topological" mapping and take the state discretisation/info from a mechanicalMapping like `TubularMapping `or `AdaptiveBeamLinearMapping`

Reply:
but I keep this in mind when I will dig into the mechanicalmapping 🤣